bzzeth/bzzeth_test.go

```diff
@@ -97,19 +114,11 @@ func TestBzzEthHandshake(t *testing.T) {
 		t.Fatalf("expected no error, got %v", err)
 	}
 
-	// after successful handshake, expect peer added to peer pool
-	var p *Peer
-	for i := 0; i < 10; i++ {
-		p = b.peers.get(node.ID())
-		if p != nil {
-			break
-		}
-		time.Sleep(100 * time.Millisecond)
-	}
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Are we ok with synchronizations with sleeps?

Reply:
How do you suggest we test this otherwise?

Reply:
With other synchronization methods, like we do in other tests. Sometimes with hooks, or events or, at least have a retry function that periodically checks for the expected condition. Sleeps are easy to do, but they are creating flaky tests where scheduler does not schedule what is expected in the expected sleep time.